### DISCUSSION TITLE
LMS Crashes immediately on startup.

### DISCUSSION THREAD

**Discussion by uudruid74** [LMS-Community/slimserver#711] [General]
OK.  I'm really fed up with this thing.  Seems like anything I do I end up killing this thing to where it won't start.  I've deleted plugins (manually) and all sorts of stuff.  Currently, I have to run it manually as a user on startup.  The same command line run via "sudo" ends up with the system dying a cruel death:

> tc@Minerva:~$ sudo /usr/bin/perl /usr/local/slimserver/slimserver.pl --user tc --group staff
> [21-12-12 21:27:16.5959] main::init (390) Starting Logitech Media Server (v8.2.1, 1639114554, Fri 10 Dec 2021 08:17:55 AM CET) perl 5.032001 - arm-linux-gnueabihf-thread-multi-64int
> [21-12-12 21:27:19.4089] Slim::Utils::Misc::msg (1341) Warning: [21:27:19.4086] private subdirectory /tmp/par-7463 is unsafe (please remove it and retry your operation) at /usr/local/slimserver/CPAN/PAR/SetupTemp.pm line 120.
> [21-12-12 21:27:19.4093] PAR::SetupTemp::set_par_temp_env (52) Warning: unable to create cache directory at /usr/local/slimserver/CPAN/PAR/SetupTemp.pm line 52.
> [21-12-12 21:27:19.4182] Slim::Utils::PluginManager::shutdownPlugins (418) Warning: error running Plugins::CastBridge::Plugin->shutdownPlugin: Can't locate object method "stop" via package "Plugins::CastBridge::Squeeze2cast" at /usr/local/slimserver/Cache/InstalledPlugins/Plugins/CastBridge/Plugin.pm line 87.

Here is the log when starting it via the web interface (which runs it as root):

> [21-12-12 21:32:26.4907] main::init (390) Starting Logitech Media Server (v8.2.1, 1639114554, Fri 10 Dec 2021 08:17:55 AM CET) perl 5.032001 - arm-linux-gnueabihf-thread-multi-64int
> [21-12-12 21:32:29.5773] Slim::Utils::Misc::msg (1341) Warning: [21:32:29.5767] private subdirectory /tmp/par-7463 is unsafe (please remove it and retry your operation) at /usr/local/slimserver/CPAN/PAR/SetupTemp.pm line 120.
> [21-12-12 21:32:29.5776] PAR::SetupTemp::set_par_temp_env (52) Warning: unable to create cache directory at /usr/local/slimserver/CPAN/PAR/SetupTemp.pm line 52.

Is there a fix?

**Comment by michaelherger** [comment]
From your log output I must assume you're running piCorePlayer? Did you expand the filesystem? 

The errors you're seeing seem to come from a plugin which is packaged in an old way. And it has problems writing to the `/tmp` folder. Did you make sure there are no left-overs?
